packages/libs/src/sdk/services/Solana/programs/RewardManagerClient/RewardManagerClient.ts

```diff
@@ -245,4 +311,74 @@ export class RewardManagerClient extends BaseSolanaProgramClient {
     }
     return this.rewardManagerState
   }
+
+  /**
+   * Override the sendTransaction method to provide some more friendly errors
```

Comment:
not needed for this PR but is this something that can be generalized for other programs?

Reply:
yeah I think so - I think each client should do similar things here